azure-functions-extension-blob/setup.py

```diff
@@ -3,6 +3,21 @@
 
 from setuptools import setup, find_packages
```

Comment:
For later, lets not combine blob and http changes in one PR. Just in case there is an issue with one, we would have to revert 2 libraries